ptcg_cockatrice.xml

```diff
@@ -1646,5 +1827,24 @@
 				{T}: Add {W}, {U}, or {B}.
 				Cycling {3}</text>
 		</card>
+		<card>
+			<name>Cinnabar Lab</name>
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="rare">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>{T}: Add {1}
+				{T}: Add {3}. This mana can only be used to pay for fossil abilities.</text>
```

Comment:
```suggestion
				{T}: Add {3}. Spend this mana only to activate fossil abilities.</text>
```

Reply:
Thanks, wasn't quite sure how to word this and couldn't find any good examples in Cockatrice